assignments/005-slurm-and-bwa-mem2/index2.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#SBATCH --cpus-per-task=1
+#SBATCH --time=0:15:00
+#SBATCH --mem=3G
+#SBATCH --job-name=fastp_trim_all
+#SBATCH --output=fastp_trim_all-%j.out
+#SBATCH --error=fastp_trim_all-%j.err
+
+####  Mike Antolin
+####  005-005-slurm-and-bwa-mem2/index2.sh
+
+####  Get ALPINE ready to SLURM
+module load slurm/alpine
+
+#### get a node
+acompile -h
```

Comment:
You don't need to do this inside the sbatch script.  sbatch will send the job to a compute node.